pkg/auth/context_config.go

```diff
@@ -13,6 +13,7 @@ var (
 	// sso.redhat.com token claim keys
 	alternateTenantUsernameClaim string = "preferred_username"
 	tenantUserIdClaim            string = "account_id"
+	subClaim                     string = "sub"
```

Comment:
nit, to keep it in sync with the current naming of claims:
```suggestion
	tenantSubClaim                     string = "sub"
```